services/instagram_service/main.py

```diff
@@ -0,0 +1,21 @@
+import os
+import asyncio
+from dotenv import load_dotenv
+from services.instagram_service.src import InstagramService
+from services.instagram_service.controller import instagram_Contoller
+from services.instagram_service.src import client_grpc
+
+load_dotenv()
+
+if __name__=='__main__':
+  username=os.environ.get('USER_NAME')
```

Comment:
It's crucial to manage situations where environment variables may not be set. At present, if any of these environment variables aren't set, your program will continue to run with these values set to ﻿None, potentially leading to unforeseen behaviors or errors down the line.

Try to consider reusing the AppConfig method we implemented in the GPT Context project.